projects/plugins/jetpack/sal/class.json-api-site-base.php

```diff
@@ -1057,6 +1057,29 @@ public function get_theme_slug() {
 		return get_option( 'stylesheet' );
 	}
 
+	/**
+	 * Returns a list of errors for broken themes on the site.
+	 *
+	 * @return array
+	 **/
+	public function get_theme_errors() {
+		$themes_with_errors = wp_get_themes( array( 'errors' => true ) );
+		$theme_errors       = array();
+
+		foreach ( $themes_with_errors as $theme ) {
+			$errors = $theme->errors();
+
+			if ( ! empty( $errors->get_error_messages() ) ) {
+				$theme_errors[] = array(
+					'name'   => $theme->get( 'Name' ),
+					'errors' => $errors->get_error_messages(),
```

Comment:
What do you think about sanitizing those values a tiny bit?

```suggestion
					'name'   => sanitize_title( $theme->get( 'Name' ) ),
					'errors' => (array) $errors->get_error_messages(),
```

Reply:
Thanks @jeherve! I agree with all your suggestions and should be added now. :)